assets/js/main.js

```diff
@@ -23,8 +23,15 @@ $("body").on('click', '#close-menu', function() {
   $('.header-main').removeClass('open');
 });
 
+$("body").on('click', '.close-user--menu', function() {
+  $('.user-menu').removeClass('open');
```

Comment:
It closes the menu when the user clicks on, for example, "COMPRAR". But should you also close the menu when the user clicks outside the menu?

![image](https://user-images.githubusercontent.com/1157864/53646521-250dfa00-3c1a-11e9-8425-66473c033b28.png)

Reply:
This is another thing we need to add. I add this for the current modal we need